include/mata/nfa.hh

```diff
@@ -357,6 +350,8 @@ public:
 
     auto states_number() const { return transition_relation.size(); }
 
+    //TODO: why this? Maybe we could have add_state(int state)
+    //Btw, why do we need adding states actually, we could just add states with transitions, or with initial/final states.
```

Comment:
It comes from nature of Vector. It is more effecient to do resize if you know that you will need more states than adding one item in time.